docs/release-guide.md

```diff
@@ -0,0 +1,97 @@
+# Release process
+
+This project uses [cargo-release][cargo-release] in order to prepare new releases, to tag and sign relevant git commit and to publish the resulting artifacts to [crates.io][crates-io].
```

Comment:
This project uses [cargo-release][cargo-release] in order to prepare new releases, ~~to~~ tag and sign relevant git commit, and ~~to~~ publish the resulting artifacts to [crates.io][crates-io].